tests/conftest.py

```diff
@@ -17,6 +17,16 @@
 ape.config.DATA_FOLDER = DATA_FOLDER
 
 
+@pytest.fixture(scope="session")
+def accounts():
+    return ape.accounts.test_accounts
+
+
+@pytest.fixture(scope="session")
+def chain():
```

Comment:
is this not a session scoped module? or are we not running via `ape test`?

Reply:
I am not used to plugins running the `ape_test` pytest plugin; i didnt know that would work actually.

Reply:
it should work just fine yeah haha, the plugin gets loaded naturally if it's installed, even locally installed